.circleci/config.yml

```diff
@@ -270,13 +273,14 @@ commands:
           command: |
             DEPLOY_DIR='<<parameters.deploy_dir>>'
             TCELL_DIR='<<parameters.tcell_dir>>'
+            DEPLOY_CONFIG_FILE_DIR='<<parameters.deploy_config_file_dir>>'
             rm -rf $DEPLOY_DIR
             mkdir -p $DEPLOY_DIR
             cd $DEPLOY_DIR
             cp -r ../$TCELL_DIR .
             cp ../output-config/tcell_agent.config $TCELL_DIR
             cp ../target/<<parameters.jar_base_name>>.jar .
-            cp ../src/appengine/<<parameters.jar_base_name>>.yaml .
+            cp "$DEPLOY_CONFIG_FILE_DIR/<<parameters.jar_base_name>>.yaml" .
             cp ../output-config/application.conf .
       - auth-gcp-service-account
       - run:
```

Comment:
This command is only used for back-end and housekeeping. Housekeeping appears to be fine.